include/libm2k/lidar.hpp

```diff
@@ -0,0 +1,52 @@
+#ifndef LIDAR_H
+#define LIDAR_H
+
+#include <map>
+#include <vector>
+#include <list>
+#include <libm2k/m2kglobal.hpp>
+#include <libm2k/context.hpp>
+
+using namespace std;
+
+namespace libm2k {
+	namespace contexts {
+		class LIBM2K_API Lidar : public Context {
+		public:
+			Lidar(std::string uri, iio_context* ctx, std::string name, bool sync);
+			virtual ~Lidar();
+
+			void channelEnableDisable(string channel, bool enable);
+			map<string, vector<int8_t>> readChannels(unsigned int nb_samples);
+	   
+			void setApdBias(unsigned int bias);
+			void setTiltVoltage(unsigned int voltage);
+			void laserEnable();
+			void laserDisable();		
+			void setLaserPulseWidth(long long pulse_width);
+			void setLaserFrequency(long long frequency);
+
+			void enableChannelSequencer();
+			void disableChannelSequencer();
+			void setSequencerPulseDelay(long long ns);
+
+			void setChannelSequencerOpModeManual();
+			void setChannelSequencerOpModeAuto();
+			void setChannelSequencerOrderAutoMode(const char* order);
+			void setChannelSequencerOrderManualMode(const char* order);
+    
+			// void enableTempCompensation();
```

Comment:
these methods are commented out;
if they won't be implemented ever, maybe let's remove them from here;

if they will be implemented later, it looks good if you add them when they will be implemented;

either way; i don't have a strong opinion about leaving this as-is

Reply:
Comments don't hurt anyone. I will leave them as such.

Reply:
agreed;
they are usually more about etiquette